lighthouse-core/lib/styles-helpers.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @param {!Array} stylesheets A list of stylesheets used by the page.
+ * @param {string=} propName Optional name of the CSS property to filter results
+ *     on. If propVal is not specified, all stylesheets that use the property are
+ *     returned. Otherwise, stylesheets that use the propName: propVal are returned.
+ * @param {string=} propVal Optional value of the CSS property to filter results on.
+ * @return {Array} A list of stylesheets that use the CSS property.
+ */
+function stylesheetsThatUsedProperty(stylesheets, propName, propVal) {
```

Comment:
I'm hesitant because every css audit will need to do that. We could add `stylesheetPropertyFilter` for audits to be able to call if they need it, but I think the most css audits will need to look across all sheets to find usage.